tests/files/bad-child-element.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
```

Comment:
These files seem to be handled correctly through the unit test, but when run for real in the shell, I always get:

```none
[fatal] Content is not allowed in prolog.
```

This seems like a UTF BOM problem, but these files do not have a BOM.